README.md

```diff
@@ -20,4 +150,4 @@ order of the chapters however may vary.
 
 ## Acknowledgements
 
-Shout out to Zed Shaw for the providing the template.
\ No newline at end of file
+Shout out to Zed Shaw for the providing the template.
```

Comment:
Shouldn't this be `for providing the template.`? Not sure, but current line feels somewhat wrong.